### PR TITLE
SDK-2486: Support dark mode in IDV SDK

### PR DESCRIPTION
**Pull request by artem-yoti**

### CODE REVIEW

File: yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java [hunk 1]
```diff
@@ -119,6 +138,15 @@ public String getLocale() {
         return locale;
     }
 
+    /**
+     * The dark mode
```
Comment (MrBurtyyy) [inline yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java:142]:
```suggestion
     * The dark mode option configured for the session
```

File: yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java [hunk 2]
```diff
@@ -92,6 +102,15 @@ public String getPrimaryColour() {
         return primaryColour;
     }
 
+    /**
+     * The primary colour for the dark mode
```
Comment (MrBurtyyy) [inline yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java:106]:
```suggestion
     * The primary colour for dark mode, configured for the session
```

File: yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java [hunk 3]
```diff
@@ -275,6 +316,47 @@ public Builder withLocale(String locale) {
...
+            return this;
+        }
+
+        /**
+         * Sets the dark mode to 'ON' to be used by the web/native client
+         *
+         * @return the builder
+         */
+        public Builder withDarkModeOn() {
+            this.darkMode = "ON";
```
Comment (MrBurtyyy) [inline yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java:336]: Can we move these String values into the `DocScanConstants` class please?

Comment (artem-yoti) [reply yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java]: Done, thank you

File: yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java (hunk 2 above)
Comment (artem-yoti) [reply yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java]: Changed, thank you for the suggestion

File: yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java (hunk 1 above)
Comment (artem-yoti) [reply yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SdkConfig.java]: Updated, thank you

**Review by bucky-boy — APPROVED** [review]
I pushed some minor changes @artem-yoti, please look before merging

**Comment by artem-yoti** [comment]
> I pushed some minor changes @artem-yoti, please look before merging

Your improvements look good to me, thank you

**Pull request merged**